packages/cadl-ranch-specs/http/projection/main.cadl

```diff
@@ -0,0 +1,59 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+
+@doc("Projection")
+@supportedBy("dpg")
+@scenarioService("/projection")
+namespace Projection;
+
+model Project {
+  @projectedName("json", "codegen")
+  producedBy?: string;
```

Comment:
Can we add `/**` comments to describe for each on of those properties what is expected name to be sent via json and generated in clients